README.md

```diff
@@ -88,7 +88,7 @@ Once TensorFlow's dependencies are installed, clone the `ngraph-bridge` repo:
 
         git clone https://github.com/tensorflow/ngraph-bridge.git
         cd ngraph-bridge
-        git checkout v0.18.0-rc1
+        git checkout v0.18.0-rc2
```

Comment:
our rc-2 uses ngraph rc3?

Reply:
yes. we started with ngrc1 (ngrc0 never had a tag from us)